src/services/dom/index.test.js

```diff
@@ -25,8 +25,12 @@ jest.mock('xmldom-instawork', () => {
 // Mock other dependencies
 const fetchMock = jest.fn();
 const responseTextMock = jest.fn();
+const headers = new Map();
+headers.set(Dom.HTTP_HEADERS.X_RESPONSE_STALE_REASON, 'stale-if-error');
 // $FlowFixMe
 fetchMock.mockResolvedValue({ text: responseTextMock });
+// $FlowFixMe
+fetchMock.mockResolvedValueOnce({ headers, text: responseTextMock });
```

Comment:
If I understand correctly, we mock once because the first test is the one expecting to use this mock? I think this is a bit brittle. Can this mock be created in the context of the test that needs it?

Reply:
addressed in https://github.com/Instawork/hyperview/pull/484/commits/664aefe39ad93cc1f0085b0e14ab5175a65e39bb